plugin/restore/mysql/mysql.go

```diff
@@ -40,14 +43,15 @@ func New(driver *mysql.Driver, instance *mysqlbinlog.Instance) *Restore {
 	}
 }
 
-// RestoreBinlog restores the database using incremental backup in time range of [config.Start, config.End).
-func (r *Restore) RestoreBinlog(ctx context.Context, config mysql.BinlogInfo) error {
+// replayBinlog restores the database using incremental backup in time range of [config.Start, config.End).
```

Comment:
There is not Start, End in the BinlogInfo

Reply:
Also since we are already under mysql module, we can just say binlog instead of incremental backup

Reply:
+1 Let's avoid talking about incremental backup which isn't specific and leads to confusion.

// replayBinlog replays the binary logs in the range of [config.Start, config.End) for a specific database.

Reply:
This is not the scope of this PR.

Reply:
Should revisit this in https://github.com/bytebase/bytebase/pull/1389 @h3n4l